src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs

```diff
@@ -0,0 +1,485 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Bit.Client.Web.BlazorUI.Components.ChoiceGroup;
+using Bit.Client.Web.BlazorUI.Playground.Web.Models;
+using Bit.Client.Web.BlazorUI.Playground.Web.Pages.Components.ComponentDemoBase;
+
+namespace Bit.Client.Web.BlazorUI.Playground.Web.Pages.Components.ChoiceGroup
+{
+    public partial class BitChoiceGroupDemo
+    {
+        private List<BitChoiceGroupOption> Example_1_5_6_Options;
+
+        private List<BitChoiceGroupOption> Example2Options;
+
+        private List<BitChoiceGroupOption> Example3Options;
+
+        private List<BitChoiceGroupOption> Example4Options;
+
+        private List<BitChoiceGroupOption> Example7Options;
+
+        /// <summary>
+        /// BitChoiceGroupDemo class constructure
+        /// </summary>
+        public BitChoiceGroupDemo()
```

Comment:
Maybe better to set the variable at the time of definition and outside the class constructor. This is easier for example. Plz check it by @msynk

Reply:
yeah, only the ones that need to use instance members of the class should be defined in a constructor.